spec/services/doc_auth/mock/doc_auth_mock_client_spec.rb

```diff
@@ -7,7 +7,8 @@
     expect(
```

Comment:
I'm not sure about this, the test name doesn't seem to match what you're testing anymore.